tables/bRO/itemslotcounttable.txt

```diff
@@ -1179,7 +1179,6 @@
 19315#1#
 19316#1#
 19326#1#
-20266#1#
```

Comment:
:thinking:

Reply:
some items show wrong slots number in bRO, maybe this is removing because this item dont have slot

Reply:
:ok_hand: